src/main/java/bio/terra/service/dataset/DatasetTable.java

```diff
@@ -24,6 +24,7 @@
     private List<Column> columns = Collections.emptyList();
     private List<Column> primaryKey = Collections.emptyList();
     private BigQueryPartitionConfigV1 bqPartitionConfig;
+    private Long rowCount;
```

Comment:
Why should we maintain a row count member on the dataset table? It is never set.

Reply:
I was just speaking with Ruchi about this -- I think we do want to set this in the near future. Inside of the query builder we query BQ for row counts when we display tables. It'd be a big time+cost savings to compute and save these at ingest + soft/hard delete times.